aladdin.sh

```diff
@@ -30,6 +31,15 @@ DEFAULT_MINIKUBE_MEMORY=4096
 
 source "$SCRIPT_DIR/shared.sh" # to load _extract_cluster_config_value
 
+# If $MANAGE_MINIKUBE is true, then all commands will be executed
+# using the external 'minikube' command.
+# https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#index-command
+# Otherwise calls to minikube will be no-op
+function minikube() {
```

Comment:
I am a bit confused, we're adding this option but then we're also doing `if "$MANAGE_MINIKUBE";` too, do we need to do both?